### ISSUE TITLE
Nightly build debugger error with NextJS 10

### ISSUE THREAD

**Issue by hect1c** [microsoft/vscode-js-debug#939]
**Describe the bug**

Following NextJS debugging: https://nextjs.org/docs/advanced-features/debugging#using-the-debugger-in-visual-studio-code. I had setup the debugger to attach to my node instance. This was working fine but in a recent release, can't remember which one this is no longer working and when I start the debugger I keep getting the following error and it crashes (not starting up) 

```
Error processing attach: Error: Could not connect to debug target at http://localhost:9229: Promise was canceled
at Object.e [as retryGetWSEndpoint] (/Users/hartley/.vscode/extensions/ms-vscode.js-debug-nightly-2021.3.817
```

This is most likely related to something in my project or configuration but it does not provide me enough information to debug. 

Below is my `launch.json` config file

```
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "attach",
      "trace": true,
      "name": "Launch Program",
      "skipFiles": ["<node_internals>/**"],
      "port": 9229
    }
  ]
}
```

Also note when I set `debug.javascript.usePreview` to `false` it works to some extent, in that my server side `debugger` is no longer caught as it used to with NextJS but it does start and I can see the console statements in the debug console.

Please advise, thanks.
 
**To Reproduce**
I have not done this using a new built instance of nextjs this is in my current project which I can't share. If still needed I will try and see if I can reproduce this in a clean install. 

**Log File**
Here is the log file: [vscode-debugadapter-dd2ec183.json.gz](https://github.com/microsoft/vscode-js-debug/files/6112285/vscode-debugadapter-dd2ec183.json.gz)

**VS Code Version:** 
Version: 1.54.1
Commit: f30a9b73e8ffc278e71575118b6bf568f04587c8

**Additional context**

**Comment by connor4312** [comment]
It looks like we just time out making the request to the process on 9229. Check to make sure that the program is launched correctly with that inspect port, and that you can access the address in your browser. If you have any firewalls or antivirus running locally, check to make sure those aren't interfering with the connection.

**Comment by hect1c** [comment]
Hey thanks for the quick response seems I was able to get it working again as you pointed me in the right direction

I had this in my `npm run dev` command `NODE_OPTIONS='--inspect --max-http-header-size=30000 --max-old-space-size=8192' next dev`.

So I decided to just remove the extra arguments and just leave `--inspect` and restart it. It worked fine and then I re-added the arguments and it's still working. I am not sure if having add that previously may have caused some issue or connection when starting up the debugger, but that seems to have resolved the issue. Thanks a lot !!
